fdbserver/Status.actor.cpp

```diff
@@ -2150,6 +2150,20 @@ ACTOR Future<JsonBuilderObject> lockedStatusFetcher(Reference<AsyncVar<CachedSer
 	return statusObj;
 }
 
+ACTOR Future<Optional<Value>> getActivePrimaryDC(Database cx) {
+	state ReadYourWritesTransaction tr(cx);
```

Comment:
This transaction should have a timeout, recording a message if we can get the info. Also, we should probably run it at immediate priority so that it still works when ratekeeper is clamped down.

Reply:
Done

Reply:
I realize now this was a little vague. The message I referred to would be to place something in the `messages` array, which would then show up in the JSON and would get printed by fdbcli when you run status. For an example, see the `loadConfiguration` function.

Reply:
Done.